pkg/controller/controller.go

```diff
@@ -22,17 +22,47 @@ package controller
 
 import (
 	"errors"
+	"fmt"
 	"sync"
 	"time"
 
 	"github.com/m3db/m3db-operator/pkg/apis/m3dboperator"
 	myspec "github.com/m3db/m3db-operator/pkg/apis/m3dboperator/v1"
+	clientset "github.com/m3db/m3db-operator/pkg/client/clientset/versioned"
+	samplescheme "github.com/m3db/m3db-operator/pkg/client/clientset/versioned/scheme"
+	informers "github.com/m3db/m3db-operator/pkg/client/informers/externalversions"
+	clusterlisters "github.com/m3db/m3db-operator/pkg/client/listers/m3dboperator/v1"
+
```

Comment:
nit: remove line

Reply:
I kind of wanna keep these separated by package since there's so many imports with the same prefix. Looking through them when they're not separated can be tough to read all them. Thoughts even if it's a bit unconventional?